docs/src/formulation-details.md

```diff
@@ -0,0 +1,85 @@
+# Formulation Details
+
+This section provides references to understand the formulations as provided by PowerModels. The list is not meant as a literature discussion, but to give the main starting points to understand the implementation of the formulations.
+
+The history of AC OPF formulations up to 2012 is discussed in:
+- Cain, M. B., O’ Neill, R. P., & Castillo, A. (2012). History of optimal power flow and formulations. <https://www.ferc.gov/industries/electric/indus-act/market-planning/opf-papers/acopf-1-history-formulation-testing.pdf>
+
+A handbook for developing an understanding of the principles of convex relaxation applied to optimal power flow is:
+- Taylor, J. A. (2015). Convex optimization of power systems (1st ed.). Cambridge University Press.
+
+## Notes on the mathematical model for all formulations
+PowerModels implements a slightly generalized version of the AC Optimal Power Flow problem from [Matpower](http://www.pserc.cornell.edu/matpower/), as discussed [here](./math-model.html).
+
```

Comment:
I believe there is a better way to do cross links within the documentation.  Check the Documentor's docs about that.

Also this is a good place to reference https://youtu.be/j7r4onyiNRQ

Reply:
done